tests/test_linux/test_wiroute.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from pyroute2.wirouting import InterfaceDoesNotExist, WiRoute, InterfaceExists
+
+pytestmark = [pytest.mark.asyncio]
+BAD_INTERFACE_NAME = "doesnotexist"
+
+
+async def test_specialized_exceptions():
```

Comment:
if possible, pls mark async tests `@pytest.mark.asyncio`

Reply:
See some lines above, the mark is done for all the file:

```
pytestmark = [pytest.mark.asyncio]
```